### PR TITLE
AZP: Fix 2 publish artefacts

### PR DESCRIPTION
**Pull request by avildema**
Fix 2 publish artefacts

### CODE REVIEW

File: buildlib/coverity.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ jobs:
             cov_error_folder=$(System.DefaultWorkingDirectory)/cov_build_${{ mode }}/output/errors
             echo "##vso[task.setvariable variable=cov_error_folder]$cov_error_folder"
             ls -la $cov_error_folder
+            mkdir -p $cov_error_folder
+            touch $cov_error_folder/index.html
```
Comment (yosefe) [inline buildlib/coverity.yml:27]: remove later?

Comment (avildema) [reply buildlib/coverity.yml:27]: sure, need for check

**Review by yosefe — CHANGES_REQUESTED** [review]
publish still fails

**Comment by avildema** [comment]
> publish still fails

success
https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=13488&view=logs&j=42a93636-9ba0-5067-13f1-66c03328bbdd&t=df4a274b-b6f3-5f2f-7dce-3379527e6707

**Comment by yosefe** [comment]
@avildema can you pls push some test commit for C code check error (for example, remove some coverity[] suppressions) to show artifacts are generated?

**Comment by avildema** [comment]
> @avildema can you pls push some test commit for C code check error (for example, remove some coverity[] suppressions) to show artifacts are generated?

@yosefe I do it direct in coverity test, link on test with artefact in top

**Comment by avildema** [comment]
> @avildema can you pls push some test commit for C code check error (for example, remove some coverity[] suppressions) to show artifacts are generated?

@yosefe https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=13514&view=artifacts&pathAsName=false&type=publishedArtifacts

all work

**Comment by yosefe** [comment]
> > @avildema can you pls push some test commit for C code check error (for example, remove some coverity[] suppressions) to show artifacts are generated?
> 
> @yosefe https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=13514&view=artifacts&pathAsName=false&type=publishedArtifacts
> 
> all work

Looks good!
but why do we have Coverity on althca? shouldn't we remove coverity from test_jenkins.sh?

**Comment by avildema** [comment]
> > > @avildema can you pls push some test commit for C code check error (for example, remove some coverity[] suppressions) to show artifacts are generated?
> > 
> > 
> > @yosefe https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=13514&view=artifacts&pathAsName=false&type=publishedArtifacts
> > all work
> 
> Looks good!
> but why do we have Coverity on althca? shouldn't we remove coverity from test_jenkins.sh?

of course , in next PR, will remove build and coverity sections, if want can remove in this PR

**Comment by yosefe** [comment]
> of course , in next PR, will remove build and coverity sections, if want can remove in this PR

sure, can be next pr

**Review by yosefe — APPROVED** [review]

**Comment by avildema** [comment]
bot:pipe:retest

**Pull request merged**